src/utils/components/PolicyForm/PolicyInterfaceExpandable.tsx

```diff
@@ -52,48 +52,54 @@ const PolicyInterfacesExpandable: FC<PolicyInterfacesExpandableProps> = ({
 
   return (
     <>
-      {policy?.spec?.desiredState?.interfaces.map((policyInterface, index) => (
-        <FormFieldGroupExpandable
-          key={`${policyInterface.type}-${index}`}
-          className="policy-interface__expandable"
-          toggleAriaLabel={t('Details')}
-          isExpanded={true}
-          header={
-            <FormFieldGroupHeader
-              titleText={{
-                text: getExpandableTitle(policyInterface, t),
-                id: `nncp-interface-${index}`,
-              }}
-              actions={
-                <Tooltip content={t('Remove interface')}>
-                  <Button
-                    variant="plain"
-                    aria-label={t('Remove')}
-                    onClick={() => removeInterface(index)}
-                  >
-                    <MinusCircleIcon />
-                  </Button>
-                </Tooltip>
+      {policy?.spec?.desiredState?.interfaces.map((policyInterface, index) => {
+        const interfaceCreated = createdInterfacesNames?.includes(policyInterface?.name);
+        return (
+          <FormFieldGroupExpandable
+            key={`${policyInterface.type}-${index}`}
+            className="policy-interface__expandable"
+            toggleAriaLabel={t('Details')}
+            isExpanded={true}
+            header={
+              <FormFieldGroupHeader
+                titleText={{
+                  text: getExpandableTitle(policyInterface, t),
+                  id: `nncp-interface-${index}`,
+                }}
+                actions={
+                  <Tooltip content={t('Remove interface')}>
+                    <span>
+                      <Button
+                        variant="plain"
+                        aria-label={t('Remove')}
+                        isDisabled={Boolean(interfaceCreated)}
+                        onClick={() => removeInterface(index)}
+                      >
+                        <MinusCircleIcon />
+                      </Button>
+                    </span>
+                  </Tooltip>
+                }
+              />
+            }
+          >
+            <PolicyInterface
+              id={index}
+              createdInterface={Boolean(
```

Comment:
I didn't see this usage here. I would rename it to `interfaceCreated` as well. I also suggest creating a function for the check you're using for this prop. It would make it look cleaner and with a good name it will help readability.